android/src/main/java/com/amazonaws/ivs/reactnative/player/AmazonIvsView.kt

```diff
@@ -481,6 +508,7 @@ class AmazonIvsView(private val context: ThemedReactContext) : FrameLayout(conte
   }
 
   fun cleanup() {
+    preloadSourceMap.clear()
```

Comment:
do you need to release all sources held here too?

Reply:
Added to the example to show the best practice, and here as well, just in case 👍